kore/test/Test/Kore/Rewrite/SMT/Symbols.hs

```diff
@@ -233,8 +230,7 @@ test_sortDeclaration =
         expr <-
             runMaybeT $
                 evalTranslator $
-                    give tools $
-                        translatePredicateWith SideCondition.top translateTerm predicate
+                    translatePredicateWith tools SideCondition.top translateTerm predicate
```

Comment:
Please break this line so it doesn't exceed 80 characters
https://github.com/runtimeverification/haskell-backend/wiki/Style-guide#line-length